package.json

```diff
@@ -13,7 +13,7 @@
     "build": "node ./node_modules/typescript/bin/tsc -b .",
     "test": "uvu -r tsm --ignore dist",
     "update-snapshots": "uvu -r tsm --ignore dist --update-snapshots",
-    "prepare": "cd snapshots && yarn && cd input/multi-project && yarn && cd ../pnpm-workspaces && pnpm install"
+    "prepare": "cd $PWD/snapshots && yarn && cd $PWD/input/multi-project && yarn && cd ../pnpm-workspaces && pnpm install"
```

Comment:
I don't quite understand how this would change anything... 🤔

Aren't all the following equivalent:

```
cd $PWD/abc
cd ./abc
cd abc
```

?

Reply:
In theory, yes, but I get the following error when running with the previous script:
```
success Already up-to-date.
$ cd snapshots && yarn && cd input/multi-project && yarn && cd ../pnpm-workspaces && pnpm install
/bin/sh: line 0: cd: snapshots: No such file or directory
error Command failed with exit code 1.
```

Are you able to reproduce, or is this something peculiar to my machine?

Reply:
Not able to reproduce. What's the output of `yarn --version` for you? I get this:

```
❯ yarn install
yarn install v1.22.19
warning ../../package.json: License should be a valid SPDX license expression
[1/4] 🔍  Resolving packages...
success Already up-to-date.
$ cd snapshots && yarn && cd input/multi-project && yarn && cd ../pnpm-workspaces && pnpm install
yarn install v1.22.19
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.04s.
yarn install v1.22.19
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.04s.
Scope: all 3 workspace projects
Lockfile is up to date, resolution step is skipped
Already up to date
Done in 299ms
✨  Done in 1.91s.
```

Is this a problem on the `main` branch on your machine? I saw you have one of Olaf's commits as the base.